ccd-definition/CaseEventToComplexTypes/uploadDocuments.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "OtherDocuments",
+    "CaseEventID": "uploadDocuments",
+    "CaseFieldID": "documents_socialWorkOther",
+    "ListElementCode": "documentTitle",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "OtherDocuments",
+    "CaseEventID": "uploadDocuments",
+    "CaseFieldID": "documents_socialWorkOther",
+    "ListElementCode": "typeOfDocument",
+    "FieldDisplayOrder": 4,
+    "DisplayContext": "OPTIONAL"
```

Comment:
Remind me why fieldDisplayOrder jumps from 1 to 4 again.

Reply:
no reason - I understand it doesn't matter?!! is that not right? Should I put 1 and 2?

Reply:
done